### DISCUSSION TITLE
I don't following anyone and you showed people that I don't want to see and don't care about in my dahsboard?

### DISCUSSION THREAD

**Discussion by buncis** [community/community#57242] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I don't following anyone and you showed people that I don't want to see and don't care about in my dahsboard?

it's distracting and useless and irrelevant 

if you want something go make something more beneficial like a review or a hype article for a rising projects

showing me people that I don't know and don't care when he staring something make me hate opening github

**Comment by buncis** [comment]
I DONT KNOW THEM I DONT CARE ABOUT HTME PLES

WHO ARE THESE STRANGERS ON MY DASHBOARD/HOMEPAGE ?

![Screenshot from 2023-06-05 22-17-11](https://github.com/community/community/assets/14258779/a1fc9a2d-bfbf-4814-9f5c-ea5da36149e0)
I have 0 following 
![image](https://github.com/community/community/assets/14258779/dc93e81b-dc77-4411-a1f9-bdc59f701a38)
Its been years and I want to complain now 
![image](https://github.com/community/community/assets/14258779/f1ca7de5-a64a-40b2-aecb-08ca79018bf6)

**Reply by buncis** [reply]
soo apparently its showed because I collaborated with them?
seriously no hard feelings for the people but I don't care with human being when I opening github its not a twitter, linkedin

**Comment by fenilli** [comment]
You can use the filter to disable all recomendations
![image](https://github.com/community/community/assets/14954720/b34abcf9-500f-4f9f-8c9c-8ab437a959a9)

**Reply by Martmists-GH** [reply]
This should be the default

EDIT: Even disabling recommendations, I get posts from repos, users and orgs I do not follow.
![Screenshot_20230606_195126](https://github.com/community/community/assets/16361449/534193ac-7fb7-4f49-8a91-0177ed447c3d)

**Reply by Martmists-GH** [reply]
Even worse, the reasons Github gives as to why these are shown are straight up WRONG.

![Screenshot_20230606_195335](https://github.com/community/community/assets/16361449/e31dfe63-402d-46ac-a448-743334881784)
![Screenshot_20230606_195341](https://github.com/community/community/assets/16361449/7d4946a6-6cf0-46cf-b197-e2306a531546)

**Reply by fenilli** [reply]
I do agree that it should be an option in the configuration to not even show recomendations in the first place, but as for me it works as expected when disabling all in the filter.
![image](https://github.com/community/community/assets/14954720/033aedcc-7157-4674-b42a-0f0bb24a54ba)
![image](https://github.com/community/community/assets/14954720/5e0f4b5c-662c-4563-8dc8-9824ea0603cb)

**Comment by bradanlane** [comment]
There needs to be an option for "don't show recommendations", "don't show forks", etc. Currently there is only "show less" and I still get a bunch.
